### DISCUSSION TITLE
Installing chrome extension opens a new tab and cypress loses the tests

### DISCUSSION THREAD

**Discussion by lutundra** [cypress-io/cypress#21165] [Questions and Help]
I am testing the chrome extension, and I install it through the code: 

module.exports = (on, config) => {
  on("before:browser:launch", (browser, launchOptions) => {
  launchOptions.extensions.push("absolute/path/to/unzipped/extension");
   return launchOptions;
  });
};
The extension though is opening the new tab as soon as Cypress is opened. So tests start running in main window, but the new tab (caused by extension) is opened and cypress loses it. 

https://user-images.githubusercontent.com/66305322/164465710-976da634-eaf9-4dfc-b81a-05620754a47d.mov

Is there any workaround ?

**Comment by minhlucvan** [comment]
any solution for this?

**Reply by mdfalcon104** [reply]
I got the same error. Using an extension and it opens a new tab first then never come back and test case failed then.

**Comment by zoza0503** [comment]
Same error

**Comment by NanoRoss** [comment]
same error, any fix ?

**Comment by zoza0503** [comment]
same

**Comment by testtolearn1** [comment]
same issue , please reply how to handle this

**Comment by jmosley5** [comment]
Hello! I'm running into the same issue. Any idea on how to get around this?
